package.json

```diff
@@ -9,9 +9,12 @@
     "build": "turbo run build",
     "test": "turbo run test",
     "lint": "turbo run lint",
-    "format": "turbo run format"
+    "format": "turbo run format",
+    "docs:generate": "typedoc"
   },
   "devDependencies": {
-    "turbo": "^1.4.5"
+    "turbo": "^1.4.5",
+    "typedoc": "^0.25.3",
+    "typescript": "5.2.x"
```

Comment:
apparently the change in typescript version causes a lint problem, and the typedoc version is not compatible with node v14